doc/nrf/links.txt

```diff
@@ -172,18 +172,18 @@
 
 .. ### Matter links that need updated SHA per release
 
-.. _`other controller setups`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/06523c2/src/controller
-.. _`CHIP Certificate Tool source files`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/06523c2/src/tools
-.. _`Bluetooth LE Arbiter's header file`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/src/platform/Zephyr/BLEAdvertisingArbiter.h
-.. _`LogModule enumeration`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/src/lib/support/logging/Constants.h
-.. _`Matter SDK's Logging header`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/src/lib/support/logging/CHIPLogging.h
-.. _`Factory Data Provider`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/src/platform/nrfconnect/FactoryDataProvider.h
-.. _`Matter nRF Connect platform source files`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/06523c2/src/platform/nrfconnect
-.. _`Generate factory data script`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py
-.. _`Factory data schema`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/scripts/tools/nrfconnect/nrfconnect_factory_data.schema
-.. _`Generate partition script`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/scripts/tools/nrfconnect/nrfconnect_generate_partition.py
-.. _`Matter nRF Connect scripts`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/06523c2/scripts/tools/nrfconnect
-.. _`Matter nRF Connect Kconfig`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/06523c2/config/nrfconnect/chip-module/Kconfig
+.. _`other controller setups`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/f7297d39cf/src/controller
```

Comment:
I think we have always used SHA of the newest commit pulled from upstream, not the top of our fork. Another topic is was it correct or not.

Reply:
Done, and also aligned the script for validating it.